### PR TITLE
About Page

### PR DESCRIPTION
**Pull request by HaydenBS93**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,443 @@
...
+            href="/blog.html"
+            class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+            >Blog</a
+          >
+          <a
+            href="/contact.html"
+            class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+            >Contact</a
+          >
+        </div>
```
Comment (naha92) [inline about.html:56]: Header different style than landing page, refer to index.html for correction

File: about.html [hunk 2]
```diff
@@ -0,0 +1,443 @@
...
+          </p>
+        </div>
+        <div class="mt-48">
+          <img
+            src="assets/images/tiago-rosado-cMG5qjpnsyg-unsplash.jpg"
+            alt="Introduction image"
+            class="w-[476px] h-[427px] object-cover rounded-lg"
+          />
+        </div>
+      </section>
```
Comment (naha92) [inline about.html:98]: Padding missing from all sides

File: about.html [hunk 3]
```diff
@@ -0,0 +1,443 @@
...
+              <span class="font-bold text-travel-orange">
+                Enjoy Downtime -
+              </span>
+              Lorem ipsum, dolor sit amet consectetur adipisicing elit. Maiores
+              impedit perferendis suscipit eaque, iste dolor cupiditate
+              blanditiis.
+            </p>
+          </div>
+        </div>
+      </section>
```
Comment (naha92) [inline about.html:168]: Padding missing from all sides

File: about.html [hunk 4]
```diff
@@ -0,0 +1,443 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Card Layout</title>
+        <link
+          href="https://cdn.jsdelivr.net/npm/tailwindcss@2.2.19/dist/tailwind.min.css"
+          rel="stylesheet"
+        />
+        <style>
+          /* Add custom styles here if needed */
+        </style>
+      </head>
```
Comment (naha92) [inline about.html:187]: Add to head at top of page

File: about.html [hunk 5]
```diff
@@ -0,0 +1,443 @@
...
+                      src="assets/icons/github.svg"
+                      alt="GitHub Icon"
+                      class="w-8 h-8"
+                    />
+                  </a>
+                </div>
+              </div>
+            </div>
+          </div>
+          <!-- Second row -->
```
Comment (naha92) [inline about.html:298]: First row responsiveness on large screens needs changing - currently on large (maybe v.large) screens the 3 cards default to align-left, should stay aligned center.

File: about.html [hunk 6]
```diff
@@ -0,0 +1,443 @@
...
+                      class="w-8 h-8"
+                    />
+                  </a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
```
Comment (naha92) [inline about.html:373]: Responsiveness needs changing for second row on tablet/mobile. On tablet one card should shift up to complete first row to have 3 rows (2-2-1). On mobile needs changing so fifth card moves below 4th card, currently cards stick together. 
Also missing links for github, currently all set to /yourusername

File: about.html [hunk 7]
```diff
@@ -0,0 +1,443 @@
...
+        </a>
+        <a href="https://www.instagram.com/">
+          <span class="sr-only">Instagram</span>
+          <img src="/assets/icons/instagram.svg" alt="Instagram icon" />
+        </a>
+      </div>
+      <p class="mt-10 text-center text-lg text-black">
+        &copy; 2024 TravelUp, Inc. All rights reserved.
+      </p>
+    </footer>
```
Comment (naha92) [inline about.html:441]: Styling different than landing page, refer to index.html for changes.  Missing padding all sides.

**Review by naha92 — CHANGES_REQUESTED** [review]
Comments added to about.html, Some minor changes, look to index.html on main branch for help.

File: about.html (hunk 1 above)
Comment (HaydenBS93) [reply about.html:56]: done

File: about.html (hunk 2 above)
Comment (HaydenBS93) [reply about.html:98]: done

File: about.html (hunk 3 above)
Comment (HaydenBS93) [reply about.html:168]: done

File: about.html (hunk 4 above)
Comment (HaydenBS93) [reply about.html:187]: done

File: about.html (hunk 5 above)
Comment (HaydenBS93) [reply about.html:298]: kept as desktop view for now until we refactor at the end when all pages are completed, updated all github usernames

File: about.html (hunk 6 above)
Comment (HaydenBS93) [reply about.html:373]: kept as desktop view for now until all pages are complete to then look at responsiveness, github usernames added

File: about.html (hunk 7 above)
Comment (HaydenBS93) [reply about.html:441]: done

File: about.html [hunk 8]
```diff
@@ -0,0 +1,398 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://cdn.jsdelivr.net/npm/tailwindcss@2.2.19/dist/tailwind.min.css"
+      rel="stylesheet"
+    />
+    <title>TravelUp - About</title>
```
Comment (HaydenBS93) [inline about.html:18]: Blog, not about

**Review by naha92 — APPROVED** [review]

**Pull request merged**